contributing.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to CONTRIBUTING.md
+
+First off, thanks for taking the time to contribute! ❤️
```

Comment:
I really like this doc!!

Should we ask customers/partners/SAs to contribute to a folder called "contrib" by default? To kind of delineate external contributions from the standard examples? Where we may use our judgement to promote something to the main set of examples every now and then?

Reply:
IMO that would confuse more than it's worth. Ultimately all examples should work and have some value and we can review whether or not they are well formed and in line with our expectations during PR reviews.

Reply:
I think we'll have lots of cases where we're not entirely sure about the value of a template. Where it's something really specific or weird.

The main use case of this repository should be in customers trying to learn more about DABs and then it can be helpful to differentiate between "blessed" and "non-blessed" examples. If we don't do that we need to "bless" everything. Which means we'll need to hard-reject things because their weird. Or that we accept contributions that are not going to be helpful for other customers.